README.md

```diff
@@ -45,9 +45,56 @@ Other options include:
 
 > Note: The Spectral CLI supports both YAML and JSON. 
 
-Currently, the CLI supports validation of OpenAPI 2 and 3 documents and lints them based on our default ruleset. It does not support custom rulesets at this time. Although if you want to build and run custom rulesets outside of the CLI, see [Customization](#Customization). 
+Currently, the CLI supports validation of OpenAPI 2 and 3 documents and lints them based on our default ruleset. It does not support custom rulesets at this time. Although if you want to build and run custom rulesets outside of the CLI, see [Customization](#Customization).
 
-## Customization
+### Example: Linting an OpenAPI document
+
+Spectral includes a number of ready made rules and functions for OpenAPI v2 and v3 documents.
+
+This example uses the OpenAPI v3 rules to lint a document.
+
+```javascript
+const { Spectral } = require('@stoplight/spectral');
+const { oas3Functions, oas3Rules } = require('@stoplight/spectral/rulesets/oas3');
+
+// an OASv3 document
+var myOAS = {
```

Comment:
Very very minor comment, but should we use `const`?
It doesn't change anything at all yet iit seems like most of the examples use `const`, so might be worth consistent to avoid any confusion?
```suggestion
const myOAS = {
```